scripts/check-go-version.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
+MINIMAL_VERSION=1.13
+VERSION=$( go version | grep -o '1\.[0-9|\.]*')
+
+function version { echo "$@" | awk -F. '{ printf("%d%03d%03d\n", $1,$2,$3); }'; }
+
+if [ $(version $VERSION) -lt $(version $MINIMAL_VERSION) ]; then
```

Comment:
nit. I suggest adding double quotes around command expansion. Version include dots and they may add to expansion friction ;)